packages/client/hmi-client/src/components/widgets/tera-input-field.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+	<div
+		class="input-container"
+		:class="{ error: errorMessage, disabled: disabled, focused: focused }"
+		@click="focusInput"
+	>
+		<label for="custom-input" class="input-label">{{ label }}</label>
+		<input
+			id="custom-input"
+			class="input-field"
+			ref="inputField"
+			:value="modelValue"
+			@input="updateValue"
+			:disabled="disabled"
+			@focus="onFocus"
+			@blur="onBlur"
```

Comment:
```suggestion
			:disabled="disabled"
			:value="modelValue"
			@input="updateValue"
			@focus="onFocus"
			@blur="onBlur"
```

Reply:
getting rid of disabled in favor of `useAttrs()`